src/components/edit-and-share/index.js

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
-import Ellipsis from './ellipsis';
+import { EllipsisOutlined } from '@ant-design/icons';
 import { Dropdown } from 'antd';
 import MenuDropdown from './menudropdown';
 
 import 'antd/dist/antd.css';
 import './style.css';
 
-function EditAndShare() {
-  const menu = <MenuDropdown />
-  return (
-    <Dropdown overlay={menu} trigger={['click']}>
-      <button className="ant-dropdown-link ellipsis" onClick={e => e.preventDefault()}>
-        <Ellipsis />
-      </button>
-    </Dropdown>
-  );
+class EditAndShare extends React.Component {
+  render() {
+    const menu = <MenuDropdown />
+    return (
+      <Dropdown overlay={menu} zIndex={2} >
+        <button className="ant-dropdown-link"
+          style={{ background: 'transparent', border: 'none', fontSize: '30px' }}
+          onClick={e => e.preventDefault()}>
```

Comment:
Si partes la línea para las props, intenta que todas queden en su propia línea para que se vea más ordenado. También, si estás usando un archivo de estilos, ¿por qué no pones esos estilos en una clase para ese botón?
```suggestion
        <button
          className="ant-dropdown-link" // Esto tendría los estilos que estas poniendo en la prop de style
          style={{ background: 'transparent', border: 'none', fontSize: '30px' }} // Esto ya no iría luego de ponerlos en la clase.
          onClick={e => e.preventDefault()}>
```

Reply:
Kike voy a tener que pasar devuelta los estilos a objeto porque la libreria no los toma